jax/scipy/special.py

```diff
@@ -47,6 +48,7 @@ def betainc(a, b, x):
 def digamma(x):
   x, = _promote_args_inexact("digamma", x)
   return lax.digamma(x)
+ad.defjvp(lax.digamma_p, lambda g, x: lax.mul(g, polygamma(1, x)))
```

Comment:
The rest of this file uses the `@custom_jvp` decorator. I'd prefer to follow that convention here

Reply:
The issue here is we want to implement jvp rule for `lax.digamma`, rather than `special.digamma`. Should I move those implementations to `lax/lax.py` file?

Reply:
Ah, I see. It's a weird dependency graph either way, unless we make `polygamma` a primitive. I think we can stick with your current approach.